### ISSUE TITLE
URLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed

### ISSUE THREAD

**Issue by gfyts0123** [TorchStudio/torchstudio#19]
When referring to the tutorial 2：transfer learning
Can't load the dataset CIFAR 10

it shows:
File C:\Users\Peter\TorchStudio\python\lib\urllib\request.py, line 1349, in do_open
raise URLError(err)
URLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)

The version of torchstudio: v0.9.5

**Comment by divideconcept** [comment]
I can't repro here, but seems to be the same issue as: https://stackoverflow.com/questions/27835619/urllib-and-ssl-certificate-verify-failed-error
Considering PyTorch/torchvision is doing its own cooking to download data, it seems the only solution to that issue would be to edit *C:\Users\Peter\TorchStudio\torchstudio\datasetload.py* and add these lines at the very beginning of the file:
```
import ssl
ssl._create_default_https_context = ssl._create_unverified_context
```

Let me know how it goes on your side.

**Comment by gfyts0123** [comment]
Now it works, thanks a lot~

**Comment by divideconcept** [comment]
Fixed in [TorchStudio 0.9.7](https://github.com/TorchStudio/torchstudio/releases/tag/0.9.7)
